packages/core/addon/models/delivery-rule.js

```diff
@@ -39,7 +39,7 @@ export default DS.Model.extend(Validations, {
       return { mustHaveData: false };
     }
   }),
-  format: DS.attr({ defaultValue: () => {} }),
+  format: DS.attr({ defaultValue: () => undefined }),
```

Comment:
Yes, it's a typescript rule extending [no-empty-function](https://eslint.org/docs/rules/no-empty-function).

Since it's a block it returns undefined so this is equivalent, just more clear

Reply:
Can this be:

```suggestion
  format: DS.attr(),
```